src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -112,6 +115,40 @@ private CallbackValidations() {
             : invalid("Exception case has no Id");
     }
 
+    @Nonnull
+    public static Validation<Seq<String>, CaseDetails> hasValidData(
```

Comment:
This is a general validation class, called `CallbackValidations`. This method returns validation for an exception record that the caseworker wants to attach to a case. I think we should name it something like `hasValidDetailsForAttachingToCase`. Not as nice, but leaves less room for misunderstanding.

Reply:
Done